lib/pengine/pe_output.c

```diff
@@ -13,6 +13,7 @@
 #include <crm/cib/util.h>
 #include <crm/msg_xml.h>
 #include <crm/pengine/internal.h>
+#include <pacemaker.h>
```

Comment:
libpe_status is below libpacemaker, so it can't include pacemaker.h. I think libcrmcommon is the easiest place to put the new stuff for use by both libpe_status and libpacemaker

Reply:
Yeah, this isn't going to be a problem in the next version of this PR.